upgrade.php

```diff
@@ -484,5 +484,6 @@ function getTableIndexes($table)
     echo "*** ERROR/WARNING SUMMARY ***\n";
     foreach ($errors as $error) {
         echo $error . "\n";
+        echo "\n";
```

Comment:
May be not well placed.
The intend was to add a blank line at the end when we launch the script in console.